estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,42 @@
+from odoo import models, fields ,api, exceptions
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate properties Offers"
+
+    price = fields.Float('Price')
+    status = fields.Selection(
+        string='Status',
+        default='new',
+        copy=False,
+        selection=[('accepted', 'Accepted'), ('refused', 'Refused'), ('new', 'New')])
+    partner_id = fields.Many2one('res.partner', string='Buyer', required=True)
+    property_id = fields.Many2one('estate.property', string='Estate Property', required=True)
+    validity = fields.Integer('Validity', default=7)
+    date_deadline = fields.Datetime('Deadline', compute='_compute_date_deadline', inverse='_inverse_date_deadline')
+
+    @api.depends('validity', 'create_date')
+    def _compute_date_deadline(self):
+        for record in self:
+            record.date_deadline = fields.Date.add(record.create_date, days=record.validity)
```

Comment:
`create_date` will have a value only after the record is already created so it is better to have a fallback on today to avoid crash during record creation

```suggestion
            record.date_deadline = fields.Date.add(record.create_date or fields.Date.today(), days=record.validity)
```